### PR TITLE
Add .shed.yml for data_manager_ncbi_taxonomy_sqlite

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
When I contributed the data_manager_ncbi_taxonomy_sqlite data manager I neglected to add a .shed.yml file, this corrects that omission.

File: data_managers/data_manager_ncbi_taxonomy_sqlite/data_manager/ncbi_taxonomy_sqlite_builder.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="ncbi_taxonomy_sqlite_builder" name="NCBI Taxonomy SQLite builder" tool_type="manage_data" version="@PACKAGE_VERSION@+galaxy0" python_template_version="3.5">
+<tool id="ncbi_taxonomy_sqlite_builder" name="NCBI Taxonomy SQLite builder" tool_type="manage_data" profile="18.09" version="@PACKAGE_VERSION@+galaxy1" python_template_version="3.5">
```
Comment (bgruening) [inline data_managers/data_manager_ncbi_taxonomy_sqlite/data_manager/ncbi_taxonomy_sqlite_builder.xml:1]: can we use the same ID as the toolshed repo or folder name?

Comment (pvanheus) [reply data_managers/data_manager_ncbi_taxonomy_sqlite/data_manager/ncbi_taxonomy_sqlite_builder.xml]: done

**Comment by pvanheus** [comment]
I think this latest failure might be testing infrastructure related...

**Comment by bgruening** [comment]
Restarted, green and merge :) Thanks @pvanheus!

**Pull request merged**